src/tasks/__init__.py

```diff
@@ -19,7 +19,14 @@
     "flare_australian": flare.Australian,
     "flare_ectsum": flare.ECTSUM,
     "flare_edtsum": flare.EDTSUM,
+    "flare_finfe": flare.FinFE,
```

Comment:
We need to distinguish Chinese tasks from other languages, please add zh in front of all these tasks, i.e, FinZHFE